x-pack/plugins/data_quality/public/utils/use_breadcrumbs.tsx

```diff
@@ -5,21 +5,30 @@
  * 2.0.
  */
 
-import type { ChromeStart } from '@kbn/core-chrome-browser';
+import type { ChromeBreadcrumb, ChromeStart } from '@kbn/core-chrome-browser';
 
-import { ManagementAppMountParams } from '@kbn/management-plugin/public';
 import { useEffect } from 'react';
+import { ManagementAppMountParams } from '@kbn/management-plugin/public';
+import { Integration } from '@kbn/dataset-quality-plugin/common/data_streams_stats/integration';
 
 export const useBreadcrumbs = (
-  breadcrumb: string,
+  breadcrumb: ChromeBreadcrumb[],
```

Comment:
Since this is now a set of breadcrumbs what about using the plural form here?
```suggestion
  breadcrumbs: ChromeBreadcrumb[],
```